backend/src/openarchiefbeheer/destruction/api/viewsets.py

```diff
@@ -474,6 +488,55 @@ def get_permissions(self):
         return [permission() for permission in permission_classes]
 
 
+@extend_schema_view(
+    list=extend_schema(
+        tags=["Co-Reviews"],
+        summary=_("List co-reviews"),
+        description=_(
+            "List all the co-reviews that have been made for a destruction list."
+        ),
+    ),
+    create=extend_schema(
+        tags=["Co-Reviews"],
+        summary=_("Create co-review"),
+        description=_(
+            "Create a co-review for a destruction list. "
+            "Only a user currently assigned as co-reviewer to the destruction list can create a review."
+        ),
+    ),
+)
+class DestructionListCoReviewViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    viewsets.GenericViewSet,
+):
+    serializer_class = DestructionListCoReviewSerializer
+    queryset = DestructionListCoReview.objects.all()
+    filter_backends = (DjangoFilterBackend,)
+    filterset_class = DestructionListCoReviewFilterset
+
+    def get_permissions(self):
+        if self.action == "create":
+            permission_classes = [IsAuthenticated & CanCoReviewPermission]
+        else:
+            permission_classes = [IsAuthenticated]
+        return [permission() for permission in permission_classes]
+
+    def get_queryset(self):
+        qs = super().get_queryset()
```

Comment:
I thought that maybe this could have been a filter in the filterset, but if you think that making it the default behaviour is better then we can leave it. Although the API schema then needs updating.

Reply:
🤔

Reply:
- [ ] Refactor to filterset